addons/mail/models/mail_message.py

```diff
@@ -27,7 +28,11 @@ class Message(models.Model):
     @api.model
     def _get_default_from(self):
         if self.env.user.email:
-            return formataddr((self.env.user.name, self.env.user.email))
+            try:
+                formatted_email = format_address((self.env.user.name, self.env.user.email))
+            except:
```

Comment:
Always use `except Exception:` when you want a catchall, so that you don't catch KeyboardInterrupt or other similar system errors.
Besides, could you give an example of when an exception will occur, and what exception that will be? (Perhaps the `except` clause could be more specific)